package.json

```diff
@@ -324,7 +324,7 @@
     "@metamask/snaps-rpc-methods": "^12.0.0",
     "@metamask/snaps-sdk": "^6.19.0",
     "@metamask/snaps-utils": "^9.0.1",
-    "@metamask/solana-wallet-snap": "^1.15.1",
+    "@metamask/solana-wallet-snap": "^1.16.0",
```

Comment:
This version is required, so the Snap can forward the `correlationId` during the `notify:accountCreated` event (which is required to correlate the `keyring_createAccount` and this event together on the Snap keyring, which is needed for the "internal options")